docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -1,388 +1,354 @@
 ---
-title: Create a REST client using .NET Core
-description: This tutorial teaches you about some features in .NET Core and the C# language.
-ms.date: 01/09/2020
-ms.assetid: 51033ce2-7a53-4cdd-966d-9da15c8204d2
+title: "Tutorial: Make HTTP requests in a .NET console application using C#"
+description: Learn how to make HTTP requests to a REST web service and deserialize JSON responses. This tutorial creates a .NET console and uses C#.
+ms.date: 04/21/2021
 ---
 
-# REST client
+# Tutorial: Make HTTP requests in a .NET console application using C\#
```

Comment:
Fine with me but I thought the guidelines recommended app for application?

Reply:
The VS UI has "Console Application", and [dotnet new](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#arguments) has the same. The style guide does say ["most of the time use app"](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=32507) but also says "It's OK to use application in content for a technical audience".  I'll change to app.